src/auxiliary/math.jl

```diff
@@ -7,6 +7,20 @@
 
 const TRIXI_UUID = UUID("a7f1ee26-1774-49b1-8366-f1abc58fbfcb")
 
+"""
+    Trixi.set_polyster(toggle::Bool; force = true)
+
+Toggle the usage of Polyster. By default Polyester is enabled, but it can
```

Comment:
```suggestion
Toggle the usage of [Polyester.jl](https://github.com/JuliaSIMD/Polyester.jl) for multithreading.
By default, Polyester.jl is enabled, but it can
```